nixos/modules/virtualisation/azure-image.nix

```
@@ -8,6 +8,11 @@ in
   imports = [ ./azure-common.nix ];
 
   options = {
+    virtualisation.azureImage.name = mkOption {
+      type = types.str;
+      description = "The name of the generated derivation";
```

```suggestion
      description = "The name of the generated image file";
```

The derivation name doesn't change here.